docs/products/tools/longview/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Longview
+title_meta: "Longview Product Documentation"
+description: "Longview is Linode’s system data graphing service. It tracks metrics for CPU, memory, and network bandwidth on both an aggregate and per-process basis."
+tab_group_main:
+    is_root: true
+    title: Overview
+    weight: 10
+cascade:
+    product_description: "View real-time system metrics and resource utilization to gain insight into your Linux-based cloud workloads."
+published: 2023-04-25
+aliases: ['/platform/longview/pricing/','/guides/linode-longview-pricing-and-plans/','/platform/longview/','/guides/platform/monitoring/']
+---
+
+Longview is Linode’s system data graphing service. It tracks metrics for CPU, memory, and network bandwidth on both an aggregate and per-process basis. It also provides real-time graphs that can help expose performance problems.
+
+## Platform-agnoistic
+
+The Longview client is [open source](https://github.com/linode/longview) and provides an agent that can be installed on any Linux distribution – including systems not hosted on the Linode platform.
+
+## Pro Plan
+
+Longview is free for all customers for up to ten clients. You also have the option to purchase **Longview Pro** which includes additional analytics for an added cost. Longview's free version updates every 5 minutes and provides twelve hours of data history. Longview Pro gives you data resolution at 60 second intervals, and you can view a complete history of your Linode’s data instead of only the previous 12 hours.
```

Comment:
```suggestion
Longview is free for all customers for up to ten clients. You also have the option to purchase **Longview Pro** which includes additional analytics for an added cost. Longview's free version updates every 5 minutes and provides 12 hours of data history. Longview Pro gives you data resolution at 60 second intervals, and you can view a complete history of your Linode’s data instead of only the previous 12 hours.
```